packages/desktop-client/src/components/manager/subscribe/Bootstrap.tsx

```diff
@@ -56,19 +58,25 @@ export function Bootstrap() {
 
   return (
     <View style={{ maxWidth: 450, marginTop: -30 }}>
-      <Title text="Welcome to Actual!" />
+      <Title text={t('Welcome to Actual!')} />
       <Paragraph style={{ fontSize: 16, color: theme.pageTextDark }}>
-        Actual is a super fast privacy-focused app for managing your finances.
-        To secure your data, you’ll need to set a password for your server.
+        {t(
+          'Actual is a super fast privacy-focused app for managing your finances.',
+        )}
+        {t(
+          'To secure your data, you’ll need to set a password for your server.',
+        )}
       </Paragraph>
 
       <Paragraph isLast style={{ fontSize: 16, color: theme.pageTextDark }}>
-        Consider opening{' '}
-        <Link variant="external" to="https://actualbudget.org/docs/tour/">
-          our tour
-        </Link>{' '}
-        in a new tab for some guidance on what to do when you’ve set your
-        password.
+        <Trans>
```

Comment:
Example using React components. The translatable string will contain placeholders like `<2>our tour</2>`